pllyrics/cog.py

```diff
@@ -282,7 +284,7 @@ async def _send_lyrics_messages(self, channel, exact, lyrics, response, show_aut
                         title=translated_message.format(
                             title_variable_do_not_translate=await track.title(),
                             page_variable_do_not_translate=i,
-                            name_variable_do_not_translate=await track.author(),
```

Comment:
You renamed the kwargs for the `.format` method but didn't change the variable within the translatable string